superset/assets/src/visualizations/deckgl/layers/common.js

```diff
@@ -34,12 +34,17 @@ export function fitViewport(viewport, points, padding = 10) {
 export function commonLayerProps(formData, slice) {
   const fd = formData;
   let onHover;
+  let tooltipContentGenerator;
   if (fd.js_tooltip) {
-    const jsTooltip = sandboxedEval(fd.js_tooltip);
+    tooltipContentGenerator = o => dompurify.sanitize(sandboxedEval(fd.js_tooltip)(o));
```

Comment:
```javascript
tooltipContentGenerator = dompurify.sanitize(sandboxedEval(fd.js_tooltip));
```

Reply:
I think this is right as is. We need to sanitize the output of `sandboxedEval(fd.js_tooltip)(o)`. The line is heavy though. I can make it 2 lines if you think there's too much going on.

Reply:
Oh, you're right. My bad.